prow/scripts/cluster-integration/reconciler-component-integration.sh

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o pipefail
+
+readonly RECONCILER_DIR="./reconciler"
+readonly GO_VERSION=1.17.5
+export KYMA_SOURCES_DIR="./kyma"
+
+function prereq_test() {
+  command -v node >/dev/null 2>&1 || { echo >&2 "node not found"; exit 1; }
+  command -v npm >/dev/null 2>&1 || { echo >&2 "npm not found"; exit 1; }
+  command -v jq >/dev/null 2>&1 || { echo >&2 "jq not found"; exit 1; }
+  command -v helm >/dev/null 2>&1 || { echo >&2 "helm not found"; exit 1; }
+  command -v kubectl >/dev/null 2>&1 || { echo >&2 "kubectl not found"; exit 1; }
+  command -v k3d >/dev/null 2>&1 || { echo >&2 "k3d not found"; exit 1; }
+}
+
+function load_env() {
+  ENV_FILE=".env"
+  if [ -f "${ENV_FILE}" ]; then
+    # shellcheck disable=SC2046
+    export $(xargs < "${ENV_FILE}")
+  fi
+}
+
+function install_cli() {
+  local install_dir
+  declare -r install_dir="/usr/local/bin"
+  mkdir -p "$install_dir"
+
+  local os
+  os="$(uname -s | tr '[:upper:]' '[:lower:]')"
+  if [[ -z "$os" || ! "$os" =~ ^(darwin|linux)$ ]]; then
+    echo >&2 -e "Unsupported host OS. Must be Linux or Mac OS X."
+    exit 1
+  else
+    readonly os
+  fi
+
+  pushd "$install_dir" || exit
+  curl -Lo kyma "https://storage.googleapis.com/kyma-cli-stable/kyma-${os}"
+  chmod +x kyma
+  popd
+
+  kyma version --client
+}
+
+function provision_k3d() {
+  k3d version
+  kyma provision k3d --ci
+}
+
+function ory::prepare_components_file() {
+  echo "Preparing Kyma installation with Ory and prerequisites"
+
+cat << EOF > "$PWD/components.yaml"
+defaultNamespace: kyma-system
+prerequisites:
+  - name: "cluster-essentials"
+  - name: "istio-configuration"
+    namespace: "istio-system"
+  - name: "certificates"
+    namespace: "istio-system"
+components:
+  - name: "ory"
+EOF
+}
+
+function deploy_kyma() {
+  local kyma_deploy_cmd
+  kyma_deploy_cmd="kyma deploy -p evaluation --ci"
```

Comment:
kyma cli shouldn't be used in this pipeline. we want to test how reconciler changes influence ory installation